config/initializers/statsd.rb

```diff
@@ -33,7 +33,20 @@ class << Samson
 end
 
 ActiveSupport::Notifications.subscribe("job_status.samson") do |*, payload|
-  Samson.statsd.increment "jobs.#{payload.fetch(:type)}.#{payload.fetch(:status)}"
+  tags = []
+  if payload[:deploy]
+    tags.push("project:#{payload[:deploy].project.name}", "stage:#{payload[:deploy].stage.name}")
+    if ['succeeded', 'failed', 'errored'].include?(payload[:status])
+      total_lead_time = payload[:deploy].total_lead_time
```

Comment:
how does lead time make sense if the deploy failed ? ... it's not done yet, so there is no lead-time ?